### ISSUE TITLE
Font size adjustable

### ISSUE THREAD

**Issue by saroad2** [EddLabs/eddington-gui#20]
One should be able to customize the font size. This is very important for the accessibility of the software.

**Comment by yuvalbardugo** [comment]
I'm on it

**Comment by saroad2** [comment]
Awesome!

Please let me know if you need any help.

**Comment by saroad2** [comment]
Fixed via #89
